Resources/Prototypes/Maps/phobos.yml

```diff
@@ -0,0 +1,60 @@
+- type: gameMap
+  id: phobos
+  mapName: 'phobos'

```

Comment:
'Phobos'

Reply:
Okay my mistake for not being more specific. As far as I can see, only this specific string, `mapName`, should be capitalised. Not id, and not the file's name. Look at the other map files. While blindly copying things has it's own problems, when you are making something new then looking at the existing examples is probably still your best starting point, and consistency is good